### PR TITLE
feat: new ui for voice cooperation

### PR DESCRIPTION
**Pull request by sparrowhe**

### CODE REVIEW

**Comment by zxq142857** [comment]
Does it stop displaying anything when not in collaboration mode?

**Comment by sparrowhe** [comment]
> Does it stop displaying anything when not in collaboration mode?

Yes, it's a deliberately design

File: src/plugins/voice-cooperation/components/VoiceFloatingNew/VoiceFloatingNew.tsx [hunk 1]
```diff
@@ -0,0 +1,428 @@
...
+              >
+                {members.map((member) => {
+                  return (
+                    <MemberListItem key={member.userInfo.id} {...member} onToggleMicrophone={onToggleMicrophone} />
+                  );
+                })}
+              </div>
+            </Box>
+            <div className={styles.divider}></div>
+            <Button className={styles.leaveButton} variant="ghost" onClick={onLeave}>
```
Comment (zxq142857) [inline src/plugins/voice-cooperation/components/VoiceFloatingNew/VoiceFloatingNew.tsx:418]: Internationalization has not been implemented

Comment (sparrowhe) [reply src/plugins/voice-cooperation/components/VoiceFloatingNew/VoiceFloatingNew.tsx:418]: 1

**Pull request merged**